util/otbn_build.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import logging as log
+import os
+import shlex
+import subprocess
+import sys
+from pathlib import Path
+from typing import List
+
+log.basicConfig(level=log.INFO, format="%(message)s")
+
+repo_top = (Path(os.path.dirname(__file__)) / '../').resolve()
```

Comment:
Conventionally, I think scripts call this REPO_TOP.

Also, if you're going to use pathlib, why not do Path(__file__).parent.parent.resolve()?